src/Components/Character/CharacterInfoBar.css

```diff
@@ -30,14 +63,14 @@
 }
 
 #xpCapacity {
-  --xpCapacity-width: 180px;
+  --xpCapacity-width: 85px;
   background-color: #aaa;
   width: var(--xpCapacity-width);
   clip-path: polygon(0 0, var(--xpCapacity-width) 0, calc(var(--xpCapacity-width) - 7px) 10px, 0 10px);
 }
 
 #xpQuantity {
-  --xpQuantity-width: 50px;
+  --xpQuantity-width: 40px;
   background-color: aqua;
   width: var(--xpQuantity-width);
   clip-path: polygon(0 0, var(--xpQuantity-width) 0, calc(var(--xpQuantity-width) - 7px) 10px, 0 10px);
```

Comment:
overall - there's an unnecessary padding gap on the left - should be fixed at css-level.